src/device-uptime/Dockerfile

```diff
@@ -1,16 +1,20 @@
-# Inherit from the Python Docker image
-FROM python:3.7-slim
+# syntax=docker/dockerfile:1
+
+FROM python:3.9-slim-buster as base
 
-# Copy the source code to app folder
 COPY . /app/
-# Change the working directory
 WORKDIR /app/
 
-# Install the Flask package via pip
+RUN pip install --upgrade pip setuptools wheel
 RUN pip install -r requirements.txt
-# Set "python" as the entry point
-# Set environment variables
+
+FROM base as staging
+ENV ENV=staging
+CMD ["python", "main.py", "--tenant=airqo"]
+
+FROM base as production
 ENV ENV=production
+CMD ["python", "main.py", "--tenant=airqo"]
 
-# Run the application
-CMD ["python", "main.py", "--tenant=airqo"]
\ No newline at end of file
+FROM base as uptime
+CMD ["python", "uptime.py"]
```

Comment:
@123MwanjeMike this second job should run every day at around 2 am.

Reply:
Hi @NoahNsimbe 
Is this a new job now, or its replacing the `FROM base as staging` and `FROM base as production` multi-stages

Reply:
its a new job